src/schema/rules/files/raw/channels.yaml

```diff
@@ -16,11 +16,11 @@ channels:
     acquisition: optional
     run: optional
 
-# ephys has an additional 'sample' entity
-channels__ephys:
+# cephys has an additional 'sample' entity
```

Comment:
In other contexts we have `microephys`, `ecephys`, `icephys`, now here `cephys`, which I think is used nowhere else. Wouldn't it make sense to use `microephys` here as well?

Reply:
good eye -- that is  a leftover , should be `microephys` I believe. Let me provide separate suggestion here across multiple lines